src/gengowatcher/orchestration/watcher_monitors.py

```
@@ -0,0 +1,169 @@
+            if hasattr(watcher, "_native_listener")
+            and hasattr(getattr(watcher, "_native_listener", None), "capture_interval")
+            else 0.75
+        )
+        time.sleep(capture_interval)
+
+
+__all__ = [
+    "run_email_monitor",
+    "run_website_monitor",
+    "run_native_browser_listener",
+]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)